code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -8,6 +8,7 @@
 	can_charge = 0
 	max_charges = 100 //100, 50, 50, 34 (max charge distribution by 25%ths)
 	var/variable_charges = 1
+	no_den_usage = TRUE
```

Comment:
This will still make all wands not allowed to be used in the den

Reply:
Yeah, was working on fixing that - didn't show up in the DM for some reason